data/20-gnome-initial-setup.rules

```diff
@@ -11,7 +11,7 @@ polkit.addRule(function(action, subject) {
 
     var actionMatches = (action.id === 'org.freedesktop.udisks2.filesystem-mount-system' ||
                          action.id === 'com.endlessm.Metrics.SetEnabled' ||
-                         action.id === 'com.endlessm.TestMode' ||
```

Comment:
Ah, of course, the factory mode still needs it (I had moved the invocation of the script into eos-demo-mode which was already privileged.